tests/ops/test_channel_ops.py

```diff
@@ -41,26 +41,37 @@
 class TestChannels:
     """Tests for the quantum channels"""
 
-    @pytest.mark.parametrize("ops", ch_list)
-    @pytest.mark.parametrize("p", [0, 0.1, 1])
+    @pytest.mark.parametrize(
+        "interface",
+        [
+            None,
+            pytest.param("autograd", marks=pytest.mark.autograd),
+            pytest.param("tensorflow", marks=pytest.mark.tf),
+            pytest.param("jax", marks=pytest.mark.jax),
+            pytest.param("torch", marks=pytest.mark.torch),
+        ],
+    )
+    @pytest.mark.parametrize("op", ch_list)
+    @pytest.mark.parametrize("p", [0.0, 0.1, 1.0])
     @pytest.mark.parametrize("tr_args", [[100e-6, 100e-6, 20e-9], [100e-6, 120e-6, 20e-9]])
-    def test_kraus_matrices_sum_identity(self, ops, p, tr_args, tol):
+    def test_kraus_matrices_sum_identity(self, op, p, tr_args, interface, tol):
         """Test channels are trace-preserving"""
-        if ops.__name__ == "GeneralizedAmplitudeDamping":
-            op = [ops(p, p, wires=0)]
-        elif ops.__name__ == "ResetError":
-            op = [ops(p / 2, p / 3, wires=0)]
-        elif ops.__name__ == "PauliError":
-            op = [ops("X", p, wires=0), ops("XX", p, wires=[0, 1])]
-        elif ops.__name__ == "ThermalRelaxationError":
-            op = [ops(p, *tr_args, wires=0)]
+        p = qml.math.array(p, like=interface)
+        if op.__name__ == "GeneralizedAmplitudeDamping":
+            ops = [op(p, p, wires=0)]
+        elif op.__name__ == "ResetError":
+            ops = [op(p / 2, p / 3, wires=0)]
+        elif op.__name__ == "PauliError":
+            ops = [op("X", p, wires=0), op("XX", p, wires=[0, 1])]
+        elif op.__name__ == "ThermalRelaxationError":
+            ops = [op(p, *tr_args, wires=0)]
         else:
-            op = [ops(p, wires=0)]
-        for operation in op:
+            ops = [op(p, wires=0)]
+        for operation in ops:
```

Comment:
```suggestion

        for operation in ops:
```
Good to give some space between code blocks.

Reply:
Thanks, good call! :)